packages/app/src/components/App/index.js

```diff
@@ -121,6 +130,8 @@ RainbowFirebaseApp.defaultProps = {
     reducers: {},
     middlewares: [],
     initialize: undefined,
+    errorComponent: undefined,
```

Comment:
here use by default the `packages/app/src/components/ErrorBoundary/errorMessage.js` component with a wrapper that sets the width and height of the screen